eth/fetcher/block_fetcher.go

```diff
@@ -803,7 +803,7 @@ func (f *BlockFetcher) enqueue(peer string, header *types.Header, block *types.B
 		hash, number = block.Hash(), block.NumberU64()
 	}
 	// Ensure the peer isn't DOSing us
-	count := f.queues[peer] + 1
+	count := f.queues[peer] + 1 // todo 4844 this is a random checkpoint but ensure that block has sidecars here
```

Comment:
I think it's ok to receive any msg, if the peer sends a block without blob, the importer will throw error.